scripts/jenkins/Jenkinsfile

```diff
@@ -1199,7 +1199,18 @@ def cpuCount() {
 /* Run ctest with appropriate env variables
  * @param target What target to pass to ctest
  */
-def ctest(target = "Test") {
+def ctest(target = "Test", testName = "Testname") {
+
+  /*
+  	 We disable the test `testshell_markdown_kdb_get` if the storage plugin INI is used, because it adds keys in between levels which
```

Comment:
I think the proper location to disable the test is the CMake code, otherwise the test will still fail, if someone uses INI as default storage plugin and runs the test suite locally. Sorry if that was not clear from the suggestions in [my previous comment](https://github.com/ElektraInitiative/libelektra/pull/2499#issuecomment-474394136).

Reply:
I am also happy with the solution in this PR (if it works). I think people should not get the impression that the INI plugin works flawlessly, as it does not. Afaik nobody (except us on the build server) currently runs the tests with INI as default storage. I think soon it will be even okay to completely remove the INI-as-default tests, as we decided to use YAML and not INI as future default storage (for good reasons).

Reply:
Thanks for your feedback!
First I'll try to figure out why my test still gets executed on debian-stable-full-ini. If I don't find my error, I'll look into the CMake code.

Reply:
The fix in the CMake code most likely is much easier as many such fixes already exist.

See doc/todo/TESTING for a list of disabled tests. Please also add your test into this list.